File: IT/1/Oggetti/Esercizi/Python/esercizio14.py
# file esercizio14.py

class Engine(object):
	def __init__(self, cc):
		self.__cc = cc

class Car(object):
	def __init__(self, model, vin, plate, engine):
		self.__model = model
		self.__vin = vin
		self.__plate = plate
		self.__engine = engine

File: IT/1/Oggetti/Esercizi/Java/Esercizio14.java
/* file Esercizio14.java */

class Engine {
	private int cc;

	public Engine(int cc) {
		this.cc = cc;
	}
}

class Car {
	private String model;
	private String vin;
	private String plate;
	private Engine engine;

	public Car(String model, String vin, String plate, Engine engine) {
		this.model = model;
		this.vin = vin;
		this.plate = plate;
		this.engine = engine;
	}
}

